test/e2e/testdata/enable_api_group_versions/music_v2beta2_rockband.yaml

```diff
@@ -5,9 +5,5 @@ metadata:
   annotations:
     rockbands.music.example.io/originalVersion: v2beta2
 spec:
-  # Add fields here
-  genre: '60s rock'
+  genre: "60s rock"
   numberComponents: 4
-  leadSinger: John
-  leadGuitar: George
-  drummer: Ringo
```

Comment:
was this removal intentional?

Reply:
Yes, these fields (leadSinger, leadGuitar, dummer) were only put there so that @brito-rafa's controller webhooks could set default values for API group version that didn't have those fields. We don't need them to test if a CRD and CR(s) were successfully backed up and restored.